app/views/claims/support/schools/mentors/remove.html.erb

```diff
@@ -0,0 +1,17 @@
+<% render "claims/support/primary_navigation", current: :organisations %>
```

Comment:
Could you have a look at the prototype for the page title and add that in here as well?

Reply:
Also, the index file is missing a page title -- I realise that wasn't your PR, but if it's not too much trouble, would be good to do it now rather than pick it up in a bug ticket later.